dallinger/command_line/__init__.py

```diff
@@ -517,10 +553,10 @@ def hits(app, sandbox):
     "--sandbox",
     is_flag=True,
     flag_value=True,
-    help="Is the app running in the sandbox?",
+    help="Look for HITs in the MTurk sandbox rather than the live/production environment",
```

Comment:
Perhaps this PR could add an option to expire a hit by HIT ID instead of just by experiment ID. As a convenience?

Reply:
@alecpm I added this feature!

Reply:
Yay! Looks good.